GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -75,31 +75,31 @@
 
     private static final String TAG = Log.tag(GVRViewManager.class);
 
-    private final Queue<Runnable> mRunnables = new LinkedBlockingQueue<Runnable>();
+    protected final Queue<Runnable> mRunnables = new LinkedBlockingQueue<Runnable>();
```

Comment:
I guess the reason why Tom change all these to protected is because the GVRMonoscopicViewManager is an extension of GVRViewManager and he wants to make all those visible to it. Correct me if I am wrong. Tomorrow I will double check if they need to be visible to GVRMonoscopicViewManager. Actually I am wondering if GVRMonoscopicViewManager need to be a child class of GVRViewManager, it may be better for code reuse issues.